docs/guide/custom-functions.md

```diff
@@ -262,6 +262,19 @@ MyFunctionPlugin.aliases = {
 };
 ```
 
+::: tip
+Aliases also require translations. Otherwise, they will not work.
+
+```js
+MyFunctionPlugin.translations = {
+  enGB: {
+    'MY_FUNCTION': 'MY_FUNCTION',
+    'MY_ALIAS': 'MY_ALIAS',
+  },
+};
+```
+:::
+
```

Comment:
During testing I decided that we should mention that.

Reply:
"will not work" sounds a bit harsh.

How about:

Aliases, just like functions, must have their name translated for the supported languages, even when the translation is the same as the function name.

Reply:
I went with @kirszenbaum's option